test/generators/csharp/__snapshots__/CSharpGenerator.spec.ts.snap

```diff
@@ -191,25 +191,129 @@ exports[`CSharpGenerator should render \`class\` type 2`] = `
 exports[`CSharpGenerator should render \`enum\` type $name should not be empty 1`] = `
 "public enum States {
   Texas, Alabama, California
-}"
+}
+public static class StatesExtensions {
+  public static dynamic GetValue(this States enumValue)
+  {
+    switch (enumValue)
+    {
+      case States.Texas: return \\"Texas\\";
+      case States.Alabama: return \\"Alabama\\";
+      case States.California: return \\"California\\";
```

Comment:
I think that there is a bug. If something is a string, then it should be treated as native string in language, not as wrapped string with `""` characters. I don't know why, but it is rendered as object, not as string - please check also another test cases - you have this bug in every place.

Reply:
It is just the snapshot :) They are rendered as: 
```
      case States.Texas: return "Texas";
      case States.Alabama: return "Alabama";
      case States.California: return "California";
```
Which is correct.

Reply:
and it's one of the cases, why I don't like snapshots 😄

Reply:
You just gotta get used to it 😆